.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!--
+The RIOT community cares a lot about code quality.
+Therefore, before describing what your contribution is about, we would like
+you to make sure that your modifications are compliant with the RIOT
+[coding conventions](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions).
```

Comment:
This isn't parsed as markdown, so using markdown syntax here might be more confusing than helpful.